tensorflow/python/compiler/tensorrt/test/trt_mode_test.py

```diff
@@ -110,5 +122,31 @@ def ExpectedEnginesToBuild(self, run_params):
     return ["TRTEngineOp_0"]
 
 
+class DynamicShapesTest(TrtModeTestBase):
+  """ Test with dynamic input shape.
+
+  The difference compered to ExplicitBatchTest is that we specify an input and
```

Comment:
This is a typo here. Can we change the comment to:
DynamicShapesTest is different from ExplicitBatchTest in that it uses input and output masks to change the input and output shapes to unknown shapes.